lib/journald/classes/logger.rb

```diff
@@ -58,7 +58,7 @@ def tag(**tags)
         yield self
       end
     ensure
-      tag(values) if values.any? # restore old values if block given
+      tag(**values) if values.any? # restore old values if block given
```

Comment:
Is this something which fails with ruby 2.7.x ? Can you provide some context?

Reply:
This is giving a warning:
```
lib/journald/classes/logger.rb:61: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
lib/journald/classes/logger.rb:46: warning: The called method `tag' is defined here
```

Reply:
Thank you, looks like a new release needs to be done.